### PR TITLE
update-submodule: add diagrams and refine the readme

### PR DESCRIPTION
**Pull request by NickVolynkin**
* Add two diagrams explaining the action's workflow.
* Document input variables `update_to` and `pr_description`.
* Reorder input variables according to the logic of the workflow.
* Add default values for optional variables.
* Reword descriptions of a few variables.

Readme preview: https://github.com/tarantool/actions/tree/nickvolynkin/update-submodule-readme/update-submodule#readme

### CODE REVIEW

File: update-submodule/README.md [hunk 1]
```diff
@@ -8,22 +8,62 @@ feature branch.
...
+%%{init: { 'gitGraph': {'mainBranchName': 'checkout_branch', 'showCommitLabel': true, 'diagramPadding': 30}} }%%
+gitGraph
+  commit id:"abc1234"
+  commit id:"def5678"
+  branch feature_branch
+  checkout feature_branch
+  commit id:"commit_message" tag:"updates submodule"
+```
+
+After pushing the branch, the action opens a pull request in the target repository:
```
Comment (NickVolynkin) [inline update-submodule/README.md:51]: When @foxzi adds `create_pr`, this line will change to "If..."

File: update-submodule/README.md [hunk 2]
```diff
@@ -8,22 +8,62 @@ feature branch.
...
+- `commit_user_email` — the author's email of the new commit;
+  default is `bot@tarantool.io`.
+- `commit_message` — the message for the commit; default is `Update submodule`.
+- `pr_against_branch` — the target repository branch to open a pull request 
+  against; usually the same as `checkout_branch`; default is `master`.
+- `pr_title` — the title of the pull request,
+  default is `[Auto-generated] Update submodule`.
+- `pr_description` — the description (body) of the pull request.
+
+<!-- - `create_pr` — if `true` (default), create a pull request in the target repository. -->
```
Comment (NickVolynkin) [inline update-submodule/README.md:32]: Saved for the next PR, will remove it before merging.

File: update-submodule/README.md [hunk 3]
```diff
@@ -8,22 +8,62 @@ feature branch.
...
+
+## How it works
+
+First, the action checks out the target repository and makes a `feature_branch`.
+Then is checkouts the `submodule` to `update_to` reference and makes a commit
+with provided name, email and message.
+Finally, it pushes the branch to the target repository:
+
+```mermaid
```
Comment (NickVolynkin) [inline update-submodule/README.md:41]: Preview: https://github.com/tarantool/actions/tree/nickvolynkin/update-submodule-readme/update-submodule#how-it-works

File: update-submodule/README.md [hunk 4]
```diff
@@ -8,22 +8,70 @@ feature branch.
...
+- `pr_against_branch` — the target repository branch to open a pull request 
+  against; usually the same as `checkout_branch`; default is `master`.
+- `pr_title` — the title of the pull request;
+  default is `[Auto-generated] Update submodule`.
+- `pr_description` — the description (body) of the pull request.
+
+## How it works
+
+First, the action checks out the target repository and makes a `feature_branch`.
+Then is checkouts the `submodule` to `update_to` reference and makes a commit
```
Comment (ylobankov) [inline update-submodule/README.md:35]: is -> it

Comment (NickVolynkin) [reply update-submodule/README.md]: fixed

**Review by ylobankov — APPROVED** [review]
LGTM

**Pull request merged**